polls/forms.py

```diff
@@ -0,0 +1,71 @@
+from datetime import datetime
+from django import forms
+from polls.models import UserProfile
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext_lazy as _
+
+
+class UserCreateForm(forms.Form):
+    BIRTH_YEAR_CHOICES = []
+    for x in range(1900, int(datetime.now().year) + 1):
+        BIRTH_YEAR_CHOICES.append(x)
+
+    SEX = (('M', _('Male')), ('F', _('Female')))
```

Comment:
choises zazwyczaj rozpisujesz tak(w modelach też jak coś)
SEX_MEN = 'M'
SEX_WOMEN = 'F'
SEX = (
(SEX_MEN, _('Male')),
(SEX_WOMEN, _('Female')),
)

Bo tak twedy jak później masz sobie gdzieś ustawić pole sex, to możesz to pobraćz z klasy, np. UserCreateForm.SEX_MEN, a nie sztywno przpisuwać 'M'. Ogólnie można to trzymać jaką integer.